src/components/ImageDetails/VPhotoDetails.vue

```diff
@@ -0,0 +1,323 @@
+  },
+  props: [
+    'image',
+    'breadCrumbURL',
+    'shouldShowBreadcrumb',
+    'imageWidth',
+    'imageHeight',
+    'imageType',
+    'thumbnail',
+  ],
+  setup(props, { emit }) {
+    const { router, store } = useContext()
+    const sketchFabfailure = ref(false)
+    const activeTab = ref(0)
+    const isReportFormVisible = ref(false)
+
+    const imgUrl = computed(() => {
+      return isLoaded.value ? props.image.url : props.thumbnail
+    })
+    const isLoaded = computed(() => props.image && !!props.image.url)
+    const providerName = computed(() => {
+      return props.image
+        ? store.getters['provider/getProviderName'](props.image.provider)
+        : ''
+    })
+    /** @type {ComputedRef<null|string>} */
+    const sketchFabUid = computed(() => {
+      if (props.image.source !== 'sketchfab' || this.sketchFabfailure) {
+        return null
+      }
+
+      return props.image.url
+        .split('https://media.sketchfab.com/models/')[1]
+        .split('/')[0]
+    })
+
+    const fullLicenseName = computed(() => {
+      return props.image
+        ? getFullLicenseName(props.image.license, props.image.license_version)
+        : ''
+    })
+    const licenseUrl = computed(
+      () => `${props.image.license_url}?ref=openverse`
+    )
+
+    const goBackToSearchResults = () => router.back()
+    const onImageLoad = (event) => emit('onImageLoaded', event)
+
+    const tabClass = (tabIdx, tabClass) => {
+      return {
+        [tabClass]: true,
+        'is-active': tabIdx === activeTab.value,
+      }
+    }
+    const setActiveTab = (tabIdx) => (activeTab.value = tabIdx)
+
+    const onCloseReportForm = () => (isReportFormVisible.value = false)
+    const toggleReportFormVisibility = () => {
+      isReportFormVisible.value = !isReportFormVisible.value
+    }
+
+    const sendEvent = (eventType) => {
+      const eventData = {
+        eventType,
+        resultUuid: props.image.id,
+      }
+      store.dispatch(`${USAGE_DATA}/${SEND_DETAIL_PAGE_EVENT}`, eventData)
+    }
+    const onPhotoSourceLinkClicked = () =>
+      sendEvent(DETAIL_PAGE_EVENTS.SOURCE_CLICKED)
+    const onPhotoCreatorLinkClicked = () =>
+      sendEvent(DETAIL_PAGE_EVENTS.CREATOR_CLICKED)
+
+    return {
+      imgUrl,
+      onImageLoad,
+      isLoaded,
+      sketchFabUid,
+      sketchFabfailure,
+      isReportFormVisible,
+      goBackToSearchResults,
+      linkClicked: {
+        photoSource: onPhotoSourceLinkClicked,
+        creatorLink: onPhotoCreatorLinkClicked,
+      },
+      license: {
+        url: licenseUrl.value,
+        fullName: fullLicenseName.value,
+      },
+      reportForm: {
+        close: onCloseReportForm,
+        toggleVisibility: toggleReportFormVisibility,
+      },
+      icons: {
+        chevronLeft,
+        externalLink,
+        flag,
+      },
+
+      activeTab,
+      tabClass,
+      setActiveTab,
+
+      providerName,
+    }
+  },
+}
+</script>
+
+<style lang="scss" scoped>
```

Comment:
This block could be replaced entirely with Tailwind utilities (including the `44rem` max-height, now that JIT is supported).